Client/Frontend/Browser/PageZoom/PageZoomSettingsView.swift

```diff
@@ -0,0 +1,172 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import Data
+import CoreData
+
+extension Domain: Identifiable {
+  public var id: String {
+    url ?? UUID().uuidString
+  }
+}
+
+private extension Domain {
+  var host: String {
+    URL(string: url ?? "")?.host ?? ""
+  }
+}
+
+struct PageZoomSettingsView: View {
+  @Environment(\.managedObjectContext) private var context
+  
+  @State private var domains = PageZoomSettingsView.filteredDomains(DataController.swiftUIContext)
+  
+  @State private var defaultZoomLevel = Preferences.General.defaultPageZoomLevel.value
+  
+  var body: some View {
+    List {
+      if !domains.isEmpty {
+        Section(
+          header: Text(Strings.PageZoom.specificWebsitesZoomLevelsSectionTitle)
+            .font(.footnote)
+            .foregroundColor(Color(.secondaryBraveLabel))
+            .multilineTextAlignment(.leading)
+        ) {
+          ForEach(domains) { domain in
+            HStack {
+              let url = domain.url ?? ""
+              
+              let zoomLevel = domain.zoom_level?.doubleValue ?? 1.0
+              
+              let percentageLevel = Int(zoomLevel * 100.0)
+              
+              Text(url)
+                .font(.headline)
+                .foregroundColor(.white)
```

Comment:
The foreground color's aren't updating in light mode �
![first open settings -  zoom level](https://user-images.githubusercontent.com/5314553/164576455-caee3dd0-3a3d-47ce-a447-7cbcf97eb73a.png)
![after visiting a website](https://user-images.githubusercontent.com/5314553/164576457-46f693d7-676a-4d06-9573-d7345b7d3b2b.png)
�

Reply:
Oh I am tired. This should be using`.black` or `.braveLabel` color I think 🙂

Reply:
No foregroundColor set at all would fix as well, it should adapt